### ISSUE TITLE
深度学习归一化 | FainKe

### ISSUE THREAD

**Issue by cookcv** [fainco/fainke.github.io#2]
https://fainke.com/2019/08/14/%E6%B7%B1%E5%BA%A6%E5%AD%A6%E4%B9%A0%E5%BD%92%E4%B8%80%E5%8C%96/ 

 内容包含：BatchNormalization、LayerNormalization、InstanceNorm、GroupNorm、SwitchableNorm 1.简述1.1 论文链接(1)、Batch Normalizationhttps://arxiv.org/pdf/1502.03167.pdf (2)、Layer Normalizaitonhttps://arxiv.org/pdf/16
